nrf-hal-common/src/uarte.rs

```diff
@@ -326,7 +326,11 @@ fn stop_write(uarte: &uarte0::RegisterBlock) {
 
 /// Start a UARTE read transaction by setting the control
 /// values and triggering a read task.
-fn start_read(uarte: &uarte0::RegisterBlock, rx_buffer: &mut [u8]) -> Result<(), Error> {
+fn start_read(
+    uarte: &uarte0::RegisterBlock,
+    rx_buffer: &mut [u8],
+    nbytes: usize,
```

Comment:
Why not just pass a shortened `rx_buffer` here?

Reply:
Oh, hello rust. Fixed.